config/install-config.yaml

```diff
@@ -0,0 +1,32 @@
+admin:
+  email: test
+  password: test
+  sshKey: |
+    ssh-rsa
+baseDomain: test
+clusterID: a7265676-7dc3-4ff3-8759-f2d6e3934e76
```

Comment:
This seems very specific; is it in fact needed?

Reply:
Something has to be set it. In this context it's basically a random hash

Reply:
this can just point to https://github.com/openshift/machine-api-operator/blob/master/config/kubemark-install-config.yaml instead

Reply:
Would not be better to point to some more real install config? I still prefer the aws one since it's way closer to the production deployment.

Reply:
this for the case where your are deploying on your own env in a non OpenShift context, the only assumption made by the operator is https://github.com/openshift/machine-api-operator/blob/master/config/kubemark-install-config.yaml#L8 anything else is has no purpose in this context